pygsti/circuits/circuit.py

```diff
@@ -3762,7 +3762,7 @@ def _write_q_circuit_tex(self, filename):  # TODO
         f.close()
 
 
-    def convert_to_stim_tableau_layers(self, gate_name_conversions=None, num_qubits=None):
+    def convert_to_stim_tableau_layers(self, gate_name_conversions=None, num_qubits=None, qubit_label_conversions=None):
```

Comment:
Can we add type annotations to this signature? Something like
```python
def convert_to_stim_tableau_layers(self, gate_name_conversions: Optional[dict[str,XXX]]=None, num_qubits: Optional[int]=None, qubit_label_conversions: Optional[dict[Union[int,str],int]=None) -> list[YYY]:
```
for suitable types XXX and YYY.

Reply:
Done. Ditto for rest of the type annotation requests so will mark those as resolved.